shared/src/Unison/ABT.hs

```diff
@@ -239,25 +272,33 @@ instance (Foldable f, Functor f, Eq1 f) => Eq (Term f) where
     go (Tm f1) (Tm f2) = f1 ==# f2
     go _ _ = False
 
-instance J.ToJSON1 f => ToJSON (Term f) where
```

Comment:
Well, the thing that is a little weird here is that we are stripping out the free variables set from the JSON (and binary) formats, and rederiving that info on the other end...